core/src/test/java/org/mule/processor/IdempotentRedeliveryPolicyTestCase.java

```diff
@@ -98,6 +100,9 @@ public ObjectStore answer(InvocationOnMock invocation) throws Throwable
             }
         });
         when(event.getMessage()).thenReturn(message);
+
+        IdempotentRedeliveryPolicyTestCase.serializer = SerializationTestUtils.getJavaSerializerWithMockContext();
```

Comment:
doing that interfeers with another test in this class and doing this as it is doesn't hurt since everything is a big mock